src/main/webapp/assets/feedStyle.css

```diff
@@ -32,11 +32,12 @@
   background-color: white;
   border: 1px solid #303030;
   color: #303030;
+  cursor: pointer;
 }
 
 /* Map */
-#map-placeholder {
-  background: var(--peach);
+#map {
+  background: var(--grey);
   height: 65vh;
   min-width: 100px;
   padding-bottom: 30px;
```

Comment:
I recommend using rem over px for accessibility. Here's a great article that breaks it down: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px

Reply:
The article mentions that using REM is good for things such as font sizes, where we want them to scale with browser defaults, but won't setting things such as margins and spaces to rem sizes instead of pixels mean that we might make certain divs less accessible? For example, if I have a div with `margin: Npx` vs `margin: Nrem`, then won't users with a larger default browser sizing see a smaller div because the margin has scaled as well? I might be misunderstanding, but I feel like this might complicate always advocating for `rem` over `px`.

Reply:
I tested it out and it doesn't appear to be the case. However, if you've tested the maps sizing on variety of sizes with pixels, that works too.